eval/tricks.rkt

```diff
@@ -168,14 +164,16 @@
            0))
 
      (if (*lower-bound-early-stopping*)
-         (list (cons (+ (maxlog y) (logspan x) (logspan z) x-slack)
+         (list (cons (max (+ (maxlog y) (logspan x) (logspan z) x-slack) x-slack)
                      (minlog y #:no-slack #t)) ; bounds per x
-               (cons (+ (maxlog y) (max (abs (maxlog x)) (abs (minlog x))) (logspan z) y-slack)
+               (cons (max (+ (maxlog y) (max (abs (maxlog x)) (abs (minlog x))) (logspan z) y-slack)
+                          y-slack)
                      (if (> (minlog x) 2)
                          (minlog y #:no-slack #t)
                          0))) ; bounds per y
-         (list (cons (+ (maxlog y) (logspan x) (logspan z) x-slack) 0) ; bounds per x
-               (cons (+ (maxlog y) (max (abs (maxlog x)) (abs (minlog x))) (logspan z) y-slack)
+         (list (cons (max (+ (maxlog y) (logspan x) (logspan z) x-slack) x-slack) 0) ; bounds per x
+               (cons (max (+ (maxlog y) (max (abs (maxlog x)) (abs (minlog x))) (logspan z) y-slack)
+                          y-slack)
```

Comment:
I included `max` in these formulas because sometimes 
we are sure that we need more precision on `x` or `y`. 
We know that `x` should have more precision when output interval of `pow` is `(ival 0.bf 1.bf)` - it means that `x` was close to `1` or `0` but not narrow enough - we 100% need more precision for `x`.
However, the general formula  `(+ (maxlog y) (logspan x) (logspan z) x-slack)` may end up being negative, in this case `x` will not receive any extra.
We force it actually receiving additional precision by `max` operator